pkg/compactor/bucket_compactor.go

```diff
@@ -539,8 +539,14 @@ func (c *BucketCompactor) runCompactionJob(ctx context.Context, job *Job) (shoul
 	if err = verifyCompactedBlocksTimeRanges(compIDs, toCompactMinTime.UnixMilli(), toCompactMaxTime.UnixMilli(), subDir); err != nil {
 		level.Warn(jobLogger).Log("msg", "compacted blocks verification failed", "err", err)
```

Comment:
Nit: Should probably be an error:

```suggestion
		level.Error(jobLogger).Log("msg", "compacted blocks verification failed", "err", err)
```